subsys/fb/Kconfig

```diff
@@ -13,28 +13,39 @@ menuconfig CHARACTER_FRAMEBUFFER
 
 if CHARACTER_FRAMEBUFFER
 
+
+choice
```

Comment:
If you make this a named choice, you can override defaults in a Kconfig.defconfig file. For example:
https://github.com/zephyrproject-rtos/zephyr/blob/1593de9642fc0e6509b22862b570c5c4d0618818/soc/arm/nxp_imx/rt/Kconfig.soc#L321-L323
https://github.com/zephyrproject-rtos/zephyr/blob/1593de9642fc0e6509b22862b570c5c4d0618818/boards/arm/mimxrt1050_evk/Kconfig.defconfig#L13-L15

Reply:
ok, I move the choice to Kconfig.defconfig, but keep the default choice in the subsys/fb/kconfig as this is for subsys bind.